apps/examples/nxplayer/nxplayer_main.c

```diff
@@ -142,7 +142,7 @@ static int nxplayer_cmd_help(FAR struct nxplayer_s *pPlayer, char *parg);
 static struct mp_cmd_s g_nxplayer_cmds[] = {
 #ifndef CONFIG_AUDIO_EXCLUDE_VOLUME
 #ifndef CONFIG_AUDIO_EXCLUDE_BALANCE
-	{"balance", "d%", nxplayer_cmd_balance, NXPLAYER_HELP_TEXT(Set balance percentage(< 50 % means more left))},
+	{"balance", "d%", nxplayer_cmd_balance, NXPLAYER_HELP_TEXT(Set balance percentage( < 50 % means more left))},
```

Comment:
In your previous changes, you removed space, but you added it now.
https://github.com/jeongchanKim/TizenRT/commit/ce542259ce688866e17e4e9f5b3f14d84d536c8d

What is right?
After checking coding rule, let me know.

Reply:
This causes infinite errors. Let's remove this.